app/services/authorisation.rb

```diff
@@ -37,6 +38,14 @@ def authorised?
     role_ids.include?(dfe_sign_in_service_access_role_id)
   end
 
+  def many?
+    org_api_path = "/users/#{user_id}/organisations"
+    response = JSON.parse(get_dfe_sign_in_api_response(org_api_path).body)
+    return true if response.count > 1
```

Comment:
This and the line below can be simplifed to returning the expression `response.count > 1`.

Reply:
Resolved this in 414ede9